package/MDAnalysis/coordinates/ParmEd.py

```diff
@@ -211,9 +215,21 @@ def convert(self, obj):
         atom_kwargs = []
         for atom, name, resname, xyz, vel in zip(ag_or_ts, names, resnames, 
                                                  positions, velocities):
-            akwargs = {'name': name}
-            chain_seg = {'segid': atom.segid}
-            for attrname in ('mass', 'charge', 'type',
+            try:
```

Comment:
I'll be honest, I'm not sure why this is being added here, was `ts` just not working to begin with? If so, do we even need to deprecate it (there's 0 end-user impact except from us removing a dead code path right?)

Reply:
But I want it to work (or at least not die) in 1.1 :-(

Reply:
🙃 we're going to need to test that the outcome makes sense then.. or you can burn it with fire.

Reply:
Too much work for something that's a little silly (making a template molecule with atoms called X, masses of 0, resids of 1...) and will be removed next release anyway.